sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/TransactionalBatchItemRequestOptions.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.RequestOptions;
+
+public final class TransactionalBatchItemRequestOptions {
```

Comment:
@Beta(Beta.SinceVersion.V4_7_0) missing for class and public members

Reply:
Fixed.